azure-bootstrap.yml

```
@@ -0,0 +1,169 @@
+
+jobs:
+- job: Trigger
+  displayName: Trigger Azure Dev Ops build and test pipeline
+  pool:
+    vmImage: 'ubuntu-latest'
+
+  variables:
+    AZURE_DEVOPS_ORG: nanoFramework
+    AZURE_DEVOPS_PROJECT: nanoFramework.Json
+    AZURE_DEVOPS_PIPELINE_ID: 59
+    AZURE_POOL_NAME: TestStream    
```

yamllint

[error] trailing spaces

(trailing-spaces)